src/app/cube/details/details.component.ts

```diff
@@ -11,11 +11,31 @@ import { ToastrOvenService } from 'app/shared/modules/toaster/notification.servi
 import { ModalListElement, ModalService } from 'app/shared/modules/modals/modal.module';
 import { AuthService } from 'app/core/auth.service';
 import { Rating } from '../old-details/details.component';
+import { ChangelogService } from 'app/core/changelog.service';
+import { trigger, transition, query, style, animate, stagger } from '@angular/animations';
 
 @Component({
   selector: 'clark-details',
   templateUrl: './details.component.html',
-  styleUrls: ['./details.component.scss']
+  styleUrls: ['./details.component.scss'],
+  animations: [
+    trigger('enterDetails', [
+      transition(':enter', [
+        query('[shouldanimate]', [
+          style({ opacity: 0, transform: 'translateY(-20px)' }),
+          stagger(100, [
+            animate('200ms ease', style({ opacity: 1, transform: 'translateY(0px)' }))
+          ])
+        ])
+      ])
+    ]),
+    // trigger('enterPage', [
```

Comment:
Can we get rid of the commented animation or do we need it for something in the future?